app/src/main/java/com/devshish/internship/presentation/ui/lyrics/LyricsFragment.kt

```diff
@@ -45,4 +42,16 @@ class LyricsFragment : Fragment(R.layout.fragment_lyrics) {
             }
         }
     }
+
+    override fun onCreateOptionsMenu(menu: Menu, inflater: MenuInflater) {
+        menu.clear()
```

Comment:
1. You do not need to call menu clear
2. You do not need to set click listener here, because you have special method for this 
So code could look like 
![image](https://user-images.githubusercontent.com/23121846/143432023-791ca834-d900-45fb-abbe-8b2a22365364.png)